source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -1041,17 +1397,28 @@ namespace AdaptiveCards { namespace Rendering { namespace Uwp
 
     [
         version(NTDDI_WIN10_RS1),
-        activatable(NTDDI_WIN10_RS1)
+        activatable(NTDDI_WIN10_RS1),
+#ifdef ADAPTIVE_CARDS_WINDOWS
+        contract(InternalContract, 1),
+        internal,
+        activatable(InternalContract, 1)
+#endif        
```

Comment:
Something to consider: would it be possible/better to have this whole chunk of stuff be a single define? Something like WINDOW_INTERNAL_ACTIVATABLE_CONTRACT that would add just one line here and include all of this so we don't have to repeat it on each runtime class? I realize there might need be a few different ones for activatable vs. not/runtimeclass vs. interface so it might be more trouble than it's worth, but just an idea.